packages/portalnetwork/src/networks/beacon/beacon.ts

```diff
@@ -614,6 +638,50 @@ export class BeaconLightClientNetwork extends BaseNetwork {
           toHexString(value),
         )
         break
+      case BeaconLightClientNetworkContentType.HistoricalSummaries: {
+        const summaries = HistoricalSummariesWithProof.deserialize(value)
+
+        // Retrieve Finality Update from lightclient to verify HistoricalSummaries proof is current
+        const finalityUpdate = this.lightClient?.getFinalized()
+        if (finalityUpdate === undefined) {
+          this.logger(`Unable to find finality update in order to verify Historical Summaries`)
+          // TODO: Decide whether it ever makes sense to accept a HistoricalSummaries object if we don't already have a finality update to verify against
+          return
+        } else {
+          const reconstructedState = ssz.capella.BeaconState.createFromProof({
```

Comment:
```suggestion
          const reconstructedStateMerkleTree = ssz.capella.BeaconState.createFromProof({
```
just a nit but provides more clarity i think